sovrin_client/client/wallet/wallet.py

```diff
@@ -187,35 +187,22 @@ def addLastKnownSeqs(self, identifier, seqNo):
     def getLastKnownSeqs(self, identifier):
         return self.lastKnownSeqs.get(identifier)
 
-    def getPendingTxnRequests(self, *identifiers):
-        if not identifiers:
-            identifiers = self.idsToSigners.keys()
-        else:
-            identifiers = set(identifiers).intersection(
-                set(self.idsToSigners.keys()))
-        requests = []
-        for identifier in identifiers:
-            lastTxn = self.getLastKnownSeqs(identifier)
-            op = {
-                TARGET_NYM: identifier,
-                TXN_TYPE: GET_TXNS,
-            }
-            if lastTxn:
-                op[DATA] = lastTxn
-            requests.append(self.signOp(op, identifier=identifier))
-        return requests
-
     def pendSyncRequests(self):
-        pendingTxnsReqs = self.getPendingTxnRequests()
-        for req in pendingTxnsReqs:
-            self.pendRequest(req)
+        # pendingTxnsReqs = self.getPendingTxnRequests()
+        # for req in pendingTxnsReqs:
+        #     self.pendRequest(req)
+
+        # GET_TXNS is discontinued
```

Comment:
Do we plan to support GET_TXNS or it's discontinued forever? If we plan to support, then either support or create a ticket in Batching Epic. If we don't plan to support it at all, then please remove this code.